core/src/main/java/org/mule/runtime/core/routing/outbound/AbstractMessageSequenceSplitter.java

```diff
@@ -34,7 +37,7 @@
  * @see AbstractSplitter
  */
 public abstract class AbstractMessageSequenceSplitter extends AbstractInterceptingMessageProcessor
-    implements MuleContextAware {
+    implements MuleContextAware, Acceptor {
```

Comment:
Compose instead of implement, because it's confusing that the splitter implement this, when in fact it is used to handle results from executed paths instead.